transport/ws/src/main/java/org/kaazing/gateway/transport/ws/AbstractWsBridgeSession.java

```diff
@@ -69,22 +70,22 @@
 
     protected ScheduledExecutorService scheduler;
     protected DefaultLoginResult loginResult;
-    protected final ActiveExtensions wsExtensions;
+    protected final ActiveWebSocketExtensions wsExtensions;
```

Comment:
For the class? Sure, but BTW it was your todo to rename it to ActiveWebSocketExtensions ;-).  For this field it will probably be removed in the near future.